<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vuelos App</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    MainWindow {
        background-image: url(&quot;imagenes/fondo.jpg&quot;) background-position: center;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Titulo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>800</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Bienvenido a Vuelos App</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>511</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
        QFrame {
            background-color: rgba(255, 255, 255, 0.8);
            border-radius: 15px;
            border: 2px solid #3498db;
        }
     </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="la_usu">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="text">
      <string>Usuario:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="li_usuario">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>180</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white; border-radius: 5px;</string>
     </property>
     <property name="placeholderText">
      <string>Introduce tu usuario</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_contra">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="text">
      <string>Contraseña:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="li_contra">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>180</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white; border-radius: 5px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Introduce tu contraseña</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bu_login">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QPushButton {
            background-color: #3498db;
            color: white;
            border-radius: 5px;
        }
        QPushButton:hover {
            background-color: #2980b9;
        }
      </string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bu_reg">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>440</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QPushButton {
            background-color: #2ecc71;
            color: white;
            border-radius: 5px;
        }
        QPushButton:hover {
            background-color: #27ae60;
        }
      </string>
     </property>
     <property name="text">
      <string>Registrarse</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="li_usuario_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>180</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white; border-radius: 5px;</string>
     </property>
     <property name="placeholderText">
      <string>Introduce tu usuario</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="li_usuario_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>180</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white; border-radius: 5px;</string>
     </property>
     <property name="placeholderText">
      <string>Introduce tu usuario</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="li_usuario_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>290</y>
       <width>180</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white; border-radius: 5px;</string>
     </property>
     <property name="placeholderText">
      <string>Introduce tu usuario</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="li_usuario_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>340</y>
       <width>180</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white; border-radius: 5px;</string>
     </property>
     <property name="placeholderText">
      <string>Introduce tu usuario</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="li_usuario_6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>380</y>
       <width>180</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white; border-radius: 5px;</string>
     </property>
     <property name="placeholderText">
      <string>Introduce tu usuario</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bu_salir">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>720</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
        QPushButton {
            background-color: red;
            color: white;
            border-radius: 5px;
        }
        QPushButton:hover {
            background-color: darkred;
        }
      </string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
